dpnp/dpnp_algo/dpnp_elementwise_common.py

```diff
@@ -594,7 +594,10 @@ def __call__(self, x, decimals=0, out=None, dtype=None):
                 dtype = x_usm.dtype
 
             out_usm = None if out is None else dpnp.get_usm_ndarray(out)
-            x_usm = dpt.round(x_usm * 10**decimals, out=out_usm)
+            # the output of x_usm multiplied by 10^decimals should be
```

Comment:
What is the limitation input `decimals`? As I can see, NumPy supports it as single integer or 0d array convertible to index scalar. Have we check that also?
Will that impact the propose changes?

Reply:
It works with 0d arrays as well.

```python
import numpy, dpnp
a = numpy.array([12, 13], dtype=numpy.int8)
numpy.round(a, decimals=1)
# array([12, 13], dtype=int8)

b = dpnp.array(a)
dpnp.round(b, decimals=1)
# array([12, 13], dtype=int8)

b = dpnp.array(a)
dpnp.round(b, decimals=dpnp.array(1))
# array([12, 13], dtype=int8)
```

Also added new tests to `test_sycl_queue.py` and `test_usm_type.py`.